static/js/keydown_util.js

```diff
@@ -3,11 +3,15 @@
 */
 
 const keys = {
-    13: "enter_key",
-    37: "left_arrow",
-    38: "up_arrow",
-    39: "right_arrow",
-    40: "down_arrow",
+    13: "enter_key", // Enter
+    37: "left_arrow", // // left arrow
+    38: "up_arrow", // up arrow
+    39: "right_arrow", // right arrow
+    40: "down_arrow", // down arrow
+    72: "vim_left", // 'H'
+    74: "vim_down", // 'J'
+    75: "vim_up", // 'K'
+    76: "vim_right", // 'L'
```

Comment:
Why do these values differ from the ones in `hotkey.js`?

Reply:
That's because `hotkeys.js` is binded with an `keypress` event and `keydown_util.js` with a `keydown` event. 
The keycodes differ by the type of event its called upon.
`keydown` displays the keycode of <kbd>a</kbd> as 65 (Similar for <kbd>A</kbd>)
`keypress` displays the keycode of <kbd>a</kbd> as 97 and <kbd>A</kbd> as 65. 

Here's a <a href = 'https://stackoverflow.com/questions/11030532/keypress-and-keyup-why-is-the-keycode-different'>stackoverflow thread</a> I found regarding the same. 

Thoughts @timabbott ?

Reply:
Maybe its better to add a comment to `keydown_util.js` explaining the same ?

Reply:
Yeah, that sounds like the right plan.